renovate-config/default.json

```diff
@@ -0,0 +1,41 @@
+{
+  "extends": [
+    "config:recommended",
+    "helpers:pinGitHubActionDigests",
+    "schedule:nonOfficeHours"
+  ],
+  "labels": [
+    "area/dependencies"
+  ],
+  "lockFileMaintenance": {
+    "enabled": true,
+    "schedule": ["before 4am on monday"],
+    "automerge": true
+  },
+  "major": {
+    "automerge": false
+  },
+  "minor": {
+    "automerge": false
+  },
+  "packageRules": [
```

Comment:
I think this packageRules could be simplified further (I'm thinking of  https://github.com/kubewarden/pod-privileged-policy/blob/main/renovate.json).

Reply:
But for our major components we usually [group all patch version](https://github.com/kubewarden/policy-server/pull/1010) bump into a single PR. If we remove that, renovate bot will open PR for every single patch update.

Reply:
I understand now! Indeed, that's better. Resolving.